components/channel_notifications_modal/components/describe.jsx

```diff
@@ -9,14 +9,20 @@ import {IgnoreChannelMentions, NotificationLevels, NotificationSections} from 'u
 
 export default function Describe({section, isCollapsed, memberNotifyLevel, globalNotifyLevel, ignoreChannelMentions}) {
     if (memberNotifyLevel === NotificationLevels.DEFAULT && globalNotifyLevel) {
+        const levelsFormattedMessageId = 'channel_notifications.levels.' + globalNotifyLevel;
         return (
-            <FormattedMessage
-                id='channel_notifications.globalDefault'
-                defaultMessage='Global default ({notifyLevel})'
-                values={{
-                    notifyLevel: (globalNotifyLevel),
-                }}
-            />
+            <React.Fragment>
+                <FormattedMessage
+                    id='channel_notifications.globalDefault'
+                    defaultMessage='Global default'
+                />
+                <span>{' ('}</span>
```

Comment:
I'm not sure, but I assume these constant parentheses shouldn't pose a problem since they don't require translation.